.github/workflows/ci.yml

```diff
@@ -216,6 +216,9 @@ jobs:
         run: cargo nextest run --workspace --locked --all-features
         env:
           VELLO_CI_GPU_SUPPORT: ${{ matrix.gpu }}
+          # If LFS downloading failed, we still want CI to work.
+          # Any values still in the cache will still work.
+          VELLO_SKIP_LFS_SNAPSHOTS: all
```

Comment:
Has this occurred already (as I haven't used such yet, but I don't think it hurts anyway)?

Reply:
I can comment this out, and say that if you need to uncomment it, to let me know and to just do so.

Reply:
I mean I'm mostly indifferent, but I think it's probably not bad to know (i.e. in failures) if this is really an issue (IME it's not).